src/audits/performance/first-meaningful-paint.js

```diff
@@ -213,7 +230,7 @@ class FirstMeaningfulPaint extends Audit {
       // layout significance = number of layout objects added / max(1, page height / screen height)
       significance = (heuristics.pageHeight) ? (layoutCount / heightRatio()) : layoutCount;
 
-      if (heuristics.webFont && counter('hasBlankText')) {
+      if (heuristics.webFont && hasTooManyBlankCharactersToBeMeaningful()) {
```

Comment:
OIC so if you're waiting on a web font and you've got a load of blank characters you're not happy. Nice!